CHANGELOG.md

```diff
@@ -1,3 +1,113 @@
+# 100.0.0
+
```

Comment:
would you be able to elaborate a bit what these improvements are? i.e. customization ability to Dialog’s animation

Reply:
Addressed.